lms/templates/enroll_staff.html

```diff
@@ -0,0 +1,40 @@
+<%inherit file="main.html" />
+<%namespace name='static' file='static_content.html'/>
+<%!
+    from django.utils.translation import ugettext as _
+    from courseware.courses import get_course_about_section
+%>
+
+<%block name="headextra">
+    <meta property="og:title" content="${course.display_name_with_default_escaped}"/>
```

Comment:
Use  `display_name_with_default ` instead of `display_name_with_default_escaped` which has been deprecated [here](http://draft-safe-templates-2016-03-23.readthedocs.io/en/latest/conventions/safe_templates.html#safe-template-linter)